install-dependencies.sh

```diff
@@ -114,20 +127,19 @@ elif [ "$DISTRO" == "debian" ]; then
    echo "Installing dependencies for Debian/Ubuntu Linux, Linux Mint..."
    sudo apt install git cmake g++ gcc gettext libasio-dev libglew-dev libpng-dev libsdl2-dev \
     libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev python zlib1g-dev
+elif [ "$DISTRO" == "solus" ]; then
+   echo "Installing dependencies for Solus..."
+   sudo eopkg install -c system.devel
+   sudo eopkg install git gettext glew-devel libicu-devel libpng-devel sdl2-devel sdl2-image-devel sdl2-mixer-devel sdl2-ttf-devel python
+   asio_not_packaged "Solus" "sudo "
 elif [ "$DISTRO" == "freebsd" ]; then
    echo "Installing dependencies for FreeBSD..."
    pkg install git asio cmake gettext glew png sdl2_image sdl2_mixer sdl2_net sdl2_ttf
 elif [ "$DISTRO" == "openbsd" ]; then
    echo "Installing dependencies for OpenBSD..."
    pkg_add git cmake gcc g++ gettext-tools glew icu4c libexecinfo png \
     sdl2-image sdl2-mixer sdl2-net sdl2-ttf
-   echo "Asio is not packaged for OpenBSD, please install it from source:"
-   echo " - Download the latest stable version from"
-   echo "      https://sourceforge.net/projects/asio/files/asio/"
-   echo " - Unpack in your development directory"
-   echo " - Run './configure --prefix=/usr && make install-data' in the unpacked asio"
-   echo "   directory (or just copy recursively 'asio.hpp' and the 'asio' subdirectory"
-   echo "   from 'include' to '/usr/include')"
+   asio_not_packaged "OpenBSD"
```

Comment:
```suggestion
   asio_not_packaged "OpenBSD" "doas "
```

According to https://www.openbsdhandbook.com/openbsd_for_linux_users/#sudo---doas …